src/frontend/next/src/pages/search.tsx

```diff
@@ -0,0 +1,62 @@
+import { FormEvent, useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { withRouter, NextRouter } from 'next/router';
+
+import SearchResults from '../components/SearchResults';
```

Comment:
Let's at least stub this out with a `<div>Search Results</div>` in `components/SearchResults` so we can land this.